Add posthog session recording for blueprints page

wryonik opened the pull request.

wryonik committed:
Add posthog session recording for blueprints page

wryonik committed:
Add example env file

wryonik commented:
Add comments explaining reset and disable_session_recording

Divide-By-0 commented:
Can you also add a small note somewhere in create blueprint first page that blueprint creation processes may be shared with the team for improvement purposes?

wryonik committed:
chore: Update posthog analytics to capture more events

- On clicking "Submit Blueprint" button
- While uploading eml and capturing event for successful/failed eml
  files
- Page views
- Time spent on the create blueprints page
- AI regex generation clicks and generated output
- Add a banner to disable analytics collection in create blueprints page
